src/styles/main.scss

```diff
@@ -1,3 +1,245 @@
-body {
+$main-text-color: #060b35;
+$text-color: #616070;
+$page-font: "Roboto", sans-serif;
+$color-blue: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+h1,
+div,
+nav,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &-logo {
+    height: 40px;
+    width: 40px;
+    align-self: center;
+  }
+}
+
+.header-nav__list {
+  display: flex;
+  margin-top: 20px;
+}
+
+.header-nav__item {
+  list-style-type: none;
+  margin-right: 20px;
+}
+
+.header-nav__item:last-child {
+  margin-right: 0;
+}
+
+.header-nav__link {
+  text: {
+    decoration: none;
+    transform: uppercase;
+  }
+  font: {
+    family: $page-font;
```

Comment:
No need. This styles are inherited from parent. You can write them in one parent.

Reply:
I hope I fixed everything)